e2e-tests/playwright/e2e/plugins/rbac/rbac-api.spec.ts

```diff
@@ -23,9 +27,18 @@ test.describe("Test RBAC plugin REST API", () => {
     responseHelper = new Response(apiToken);
   });
 
-  test.beforeEach(
-    async () => await new Common(page).checkAndClickOnGHloginPopup(),
-  );
+  // eslint-disable-next-line no-empty-pattern
+  test.beforeEach(async ({}, testInfo) => {
+    try {
+      console.log(
+        `beforeEach: Attempting setup for ${testInfo.title}, retry: ${testInfo.retry}`,
+      );
+      await new Common(page).checkAndClickOnGHloginPopup();
+    } catch (error) {
+      console.error(`beforeEach failed for ${testInfo.title}:`, error);
+      throw error; // Ensure Playwright handles the failure correctly for retries
```

Comment:
if you throw the error from catch its the same logic without the try-catch block, try catch may not be needed

Reply:
I agree that throwing the error directly could simplify the code. I'll adjust it to remove unnecessary try-catch blocks.